apps/examples/testcase/ta_tc/device_management/itc/itc_dm_lwm2m.c

```diff
@@ -16,46 +16,18 @@
  *
  ****************************************************************************/
 
-#include <tinyara/config.h>
 #include <stdio.h>
-#include <errno.h>
-#include <sys/stat.h>
-#include <net/if.h>
-#include <arpa/inet.h>
-#include <netinet/in.h>
-#include <netutils/netlib.h>
-#include <sys/socket.h>
 #include "itc_internal.h"
 #include <dm/dm_lwm2m.h>
 #include <dm/dm_error.h>
-#include <tinyara/wqueue.h>
 
 #define ITC_DM_IPADDR_LEN 20
 #define ITC_DM_SERVER_PORT 10
+#define LIFE_TIME 300
+#define SEC_1 1
 
-struct dm_lwm2m_context_s test_data_itc = { {"192.168.0.105", "5683", false}, {300} };
-static struct work_s itc_dm_wq;
+struct dm_lwm2m_context_s test_data_itc = { { CONFIG_ITC_DM_SERVER_IPADDR, CONFIG_ITC_DM_SERVER_PORT, false }, { LIFE_TIME } };
```

Comment:
If use config value directly, you had better define below to prevent error.
#ifndef CONFIG_ITC_DM_SERVER_IPADDR
#define CONFIG_ITC_DM_SERVER_IPADDR "192.168.0.105"
#endif